codelab.md

```diff
@@ -1,9 +1,9 @@
 author: Maxime LUMEAU, Robin CAROFF and Pierre TIBULLE
 summary: Kotlin Multiplatform Workshop
-id: codelab-kotlin-Multiplatform
+id: codelab-kotlin-multiplatform
 categories: codelab,kotlin
 environments: Web
-status: Draft
+status: Published
```

Comment:
@ptibulle Why do we have to set it as published ? It's a draft isn't it ?

Reply:
Yes it is ! Copy/Paste problem ???

Reply:
OK. I'll change it for Draft then.